PC/layout/support/appxmanifest.py

```diff
@@ -453,7 +453,7 @@ def get_appxmanifest(ns):
             ["pip", "pip{}".format(VER_MAJOR), "pip{}".format(VER_DOT)],
             PIP_VE_DATA,
             "console",
-            ("python.wheel", [".whl"], 'install "%1"', "Python Wheel"),
+            ("python.wheel", [".whl"], 'install "%1" %*', "Python Wheel"),
```

Comment:
I'm not sure how useful it is to have this for wheels - I wouldn't expect anyone to be trying to directly execute a `.whl` file, and if they did I doubt they'd expect to be able to supply extra pip options on the command line.

I'd only see this association as useful for people to double click a `.whl` file (or use "run this file" from a browser download, etc).

Reply:
Yeah, fair enough. `--no-deps` would be the only option I'd use myself, but then I don't use any of these associations anyway.